js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-design.js

```diff
@@ -164,10 +181,57 @@ define([
             });
         }
 
+        addStrata() {
+            const strata = {
+              name: ko.observable(),
+              criteria: ko.observable(new CriteriaGroup(null, this.conceptSets))
+				    };
+            const ccDesign = this.design();
+            this.design({
+              ...ccDesign,
+              stratas: [
+                ...(ccDesign.stratas || []),
+                strata
+              ],
+						})
+        }
+
+        removeStrata(index) {
+            const ccDesign = this.design();
+            this.design({
+              ...ccDesign,
+              stratas: ccDesign.stratas.filter((s, i) => i !== index)
+            });
+        }
 
         showParameterCreateModal() {
             this.isParameterCreateModalShown(true);
         }
+
+        handleConceptSetImport(criteriaIdx, item) {
+          console.log('import', item);
+          this.criteriaContext({...item, criteriaIdx});
+          this.showConceptSetBrowser(true);
+        }
+
+			onRespositoryConceptSetSelected(conceptSet, source) {
```

Comment:
Looks very similar to the method with same naming in `feature-analysis-view-edit`. Let's create a common class to exclude code duplication